src/main/webapp/app/admin/lti-configuration/lti-configuration.service.ts

```diff
@@ -10,8 +11,12 @@ export class LtiConfigurationService {
     /**
      * Sends a GET request to retrieve all lti platform configurations
      */
-    findAll(): Observable<LtiPlatformConfiguration[]> {
-        return this.http.get<LtiPlatformConfiguration[]>('api/lti-platforms');
+    query(req: any): Observable<HttpResponse<LtiPlatformConfiguration[]>> {
```

Comment:
I see. Under the circumstances that you utilise the very same method as similar occurrences, I think we can overlook the fact that it violates the guidelines. Once there is a common definition, it can be easily found by traversing the function calls.